kinetic/distribution.yaml

```diff
@@ -16445,6 +16445,21 @@ repositories:
       url: https://github.com/KumarRobotics/ublox.git
       version: master
     status: maintained
+  udp_com:
+    doc:
+      type: git
+      url: https://github.com/continental/udp_com.git
+      version: gh-pages
```

Comment:
I think this should be `ros1/main` instead of the static website for github. I'm not sure where it's documented, but I think the doc jobs work by building the packages and then invoking [rosdoc_lite](http://wiki.ros.org/rosdoc_lite), so `doc` `version` should probably be the same as your `source` `version`.

http://build.ros.org/job/Kdoc__roscpp_core__ubuntu_xenial_amd64/8/console

```
00:04:32.830 Invoking `rosdoc_lite` for package 'cpp_common': /tmp/rosdoc_lite/scripts/rosdoc_lite /tmp/ws/src/roscpp_core/cpp_common -o /tmp/generated_documentation/api_rosdoc/cpp_common -g /tmp/generated_documentation/symbols/cpp_common.tag -t /tmp/generated_documentation/rosdoc_tags/cpp_common.yaml
```

Reply:
```suggestion
      version: ros1/main
```

Reply:
is it ok to just commit the change on this PR or do you want me to make another bloom release? ill be sure to update the track on future releases

Reply:
It's fine to manually update the PR :)